checks/evaluation/ci_tests.go

```diff
@@ -61,8 +61,8 @@ func CITests(name string, c *checker.CITestData, dl checker.DetailLogger) checke
 		if !foundCI {
 			// Log message says commit, but really we only care about PRs, and
 			// use only one commit (branch HEAD) to refer to all commits in a PR
-			dl.Debug(&checker.LogMessage{
-				Text: fmt.Sprintf("merged PR without CI test at HEAD: %s", r.HeadSHA),
+			dl.Warn(&checker.LogMessage{
```

Comment:
is the reason for changing this because we want to have it available in the BQTable? If so, could we instead run the cron job with --verbosity debug?

@spencerschrock @azeemsgoogle wdut?

NOTE: we have to be careful how this interacts with SARIF results (for which we don't have good e2e tests atm). Every new `Warn` in the details may create a new alerts in the GitHub scanning dashboard https://github.com/ossf/scorecard/blob/main/pkg/sarif.go#L309. In the case of the code above, it won't happen because there's no "file" associated with the warning.

Reply:
I believe you want `Info` here - `Info` displays with `--show-details` but probably doesn't interfere with SARIF/Cron.

Reply:
Info is for positive things. Here we're reporting something that's negative, iiuc

Reply:
Thanks!